plugins/poll/assets/javascripts/discourse/components/poll-voters-ranked-choice.gjs

```diff
@@ -38,7 +38,13 @@ export default class PollVotersComponent extends Component {
         {{/if}}
         {{#each rank.voters as |user|}}
           <li>
-            {{avatar user.user.avatar_template "tiny"}}
+            <a data-user-card={{user.user.username}}>{{avatar
```

Comment:
Note this `user.user` is intended.  In Ranked voting, each voter includes rank information needed for results (in `user.rank`)